### ISSUE TITLE
Very Coooool!!!!

### ISSUE THREAD

**Issue by bigdimboom** [GPUOpen-LibrariesAndSDKs/FidelityFX-SDK#11]
What are the cmake flags do I use to build this and examples on linux + vulkan?

**Comment by bigdimboom** [comment]
No, worries. I will play around with it. There is awesome man!!!

**Comment by dspcad** [comment]
Hi @jlacroixAMD 

I tried to run cmake to build FidelityFX on ubuntu but got the error below:
FidelityFX-SDK/framework/cauldron/framework/libs/imgui/backends/imgui_impl_win32.cpp:20:10: fatal error: windows.h: No such file or directory
   20 | #include <windows.h>
      |          ^~~~~~~~~~~
compilation terminated.

Is there any way I can avoid windows part? Thanks.
